### PR TITLE
JW-1199: Check if extension is running after enabling

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

**Comment by krdln** [comment]
Not sure about the error message.

Should I add a comment about it being a workaround for some windows/extension bug?

**Comment by m-kostrzewa** [comment]
> Should I add a comment about it being a workaround for some windows/extension bug?

steps to repro maybe?

**Comment by krdln** [comment]
@m-kostrzewa They're in jira ticket

**Comment by m-kostrzewa** [comment]
We probably should add a test case that shows steps to repro and asserts that driver init fails

**Review by jablonskim — APPROVED** [review]

File: driver/driver.go [hunk 1]
```diff
@@ -92,6 +92,16 @@ func (d *ContrailDriver) StartServing() error {
...
 			return err
 		}
+
+		running, err := hyperv.IsExtensionRunning(d.vswitchName)
+		if err != nil {
+			return err
+		}
+
+		if !running {
+			return errors.New("Extension stopped running after enable. " +
```
Comment (mclapinski) [inline driver/driver.go]: should be `after being enabled`

**Review by mclapinski — CHANGES_REQUESTED** [review]

**Review by mclapinski — APPROVED** [review]

**Pull request merged**